pkg/node/manager/manager.go

```diff
@@ -734,6 +734,26 @@ func (m *manager) NodeUpdated(n nodeTypes.Node) {
 	m.ipsetMgr.AddToIPSet(ipset.CiliumNodeIPSetV4, ipset.INetFamily, v4Addrs...)
 	m.ipsetMgr.AddToIPSet(ipset.CiliumNodeIPSetV6, ipset.INet6Family, v6Addrs...)
 
+	if !n.IsLocal() {
```

Comment:
What about if this node is the local node? Do we also need to plumb the node pod CIDR into the ipcache?

Reply:
Not for tunneling (we don't insert local node allocation CIDRs into the tunnel map).
But it is something we can consider if any other feature can benefit from these new entries.